arch/x86/core/fatal.c

```diff
@@ -285,6 +285,8 @@ static void log_exception(uintptr_t vector, uintptr_t code)
 		LOG_ERR("Security exception");
 		break;
 	default:
+		/* Add the default case with break, this is a defensive programming. */
```

Comment:
I don't want to be pedantic, but I rather not having this kind of obvious comments in the code. I got that we have to do this in cases where the only instruction is `break`. But this is no longer the case. Imagine adding this comment in *ALL* default labels ...

Reply:
+1
```suggestion
```